src/components/context.tsx

```diff
@@ -2,9 +2,11 @@ import createReactContext, {
   ConsumerProps,
   ProviderProps,
 } from 'create-react-context';
-import { ComponentClass } from 'react';
+import { createContext, ComponentClass } from 'react';
 
-const context = createReactContext({});
+export const context = createContext
+  ? createContext({})
+  : createReactContext({});
```

Comment:
hooks does not work with `create-react-context`. So this ternary allows the runtime to use native context if it exists on the users react version.

Reply:
This is probably something that should ship in master anyhow.

Reply:
Isn't `create-react-context` supposed to do that automatically? 🤔 https://github.com/jamiebuilds/create-react-context/blob/master/src/index.js#L5